.npmrc

```diff
@@ -0,0 +1 @@
+registry=https://npm.lwcjs.org/
```

Comment:
We will use this repository so that we can access some of our own packages for now that are yet to be available.